src/Yoti.Auth/ShareUrl/Policy/SourceConstraintBuilder.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Collections.Generic;
+
+namespace Yoti.Auth.ShareUrl.Policy
+{
+    public class SourceConstraintBuilder
+    {
+        internal const string _anchorValuePassport = "PASSPORT";
+        internal const string _anchorValueDrivingLicense = "DRIVING_LICENCE";
+        internal const string _anchorValueNationalId = "NATIONAL_ID";
+        internal const string _anchorValuePassCard = "PASS_CARD";
```

Comment:
Should these be `WantedAnchor` constants?

The PHP SDK has constants such as `WantedAnchor::VALUE_PASSPORT` - should we change PHP or perhaps it doesn't matter for now?

Reply:
I have updated them to use the constants in Constants.DocumentDetails in 15a270f